codespell_lib/data/dictionary.txt

```diff
@@ -55877,6 +55877,7 @@ strory->story
 strotage->storage
 stroy->story, destroy,
 stroyboard->storyboard
+strring->string, stirring,
```

Comment:
```suggestion
strring->string, stirring, starring,
strrings->strings
```

Reply:
"starring" was a great addition, thank you!

I tried to commit directly from this review but kept getting the message "This diff has recently been updated. Refresh and try again." I did apply the correction outside the browser though.